.github/pull_request_template.md

```diff
@@ -1,23 +1,35 @@
+<!-- NOTE: Do not modify these when initially opening the pull request.  This is a checklist template that you tick off AFTER the pull request is created. -->
```

Comment:
I think this should be closer to the author checklist like it was before.  Thoughts?

Moved to the beginning, the phrase "Do not modify these" doesn't have context anymore.  The comment is just for the author, not the reviewer, and is primarily aimed at first time contributors who might try to edit the checklist markdown.